docs/permissions/getting-started.md

```diff
@@ -38,82 +26,75 @@ Like many other parts of Backstage, the permissions framework relies on informat
 
 [The IdentityResolver docs](../auth/identity-resolver.md) describe the process for resolving group membership on sign in.
 
-## Optionally add cookie-based authentication
-
-Asset requests initiated by the browser will not include a token in the `Authorization` header. If these requests check authorization through the permission framework, as done in plugins like TechDocs, then you'll need to set up cookie-based authentication. Refer to the ["Authenticate API requests"](https://github.com/backstage/backstage/blob/master/contrib/docs/tutorials/authenticate-api-requests.md) tutorial for a demonstration on how to implement this behavior.
+## Test Permission Policy
 
-## Integrating the permission framework with your Backstage instance
+To get help validate the permission framework is setup we'll create a Test Permission Policy:
 
-### 1. Set up the permission backend
+1. Backstage ships with a default Allow All Policy, we want to remove that as it would override our Test Permission Policy. To do this remove the following line:
 
-The permissions framework uses a new `permission-backend` plugin to accept authorization requests from other plugins across your Backstage instance. The Backstage backend does not include this permission backend by default, so you will need to add it:
+   ```ts title="packages/backend/src/index.ts"
+   // permission plugin
+   backend.add(import('@backstage/plugin-permission-backend/alpha'));
+   /* highlight-remove-start */
+   backend.add(
+     import('@backstage/plugin-permission-backend-module-allow-all-policy'),
+   );
+   /* highlight-remove-end */
+   ```
 
-1. Add `@backstage/plugin-permission-backend` as a dependency of your Backstage backend:
+2. Now we need to add the `@backstage/backend-plugin-api` package:
 
-   ```bash title="From your Backstage root directory"
-   yarn --cwd packages/backend add @backstage/plugin-permission-backend
+   ```bash title="from your Backstage root directory"
+   yarn --cwd packages/backend add @backstage/backend-plugin-api
    ```
 
-2. Add the following to a new file, `packages/backend/src/plugins/permission.ts`. This adds the permission-backend router, and configures it with a policy which allows everything.
+3. Next we will create an `extensions` folder under `packages/backend/src`
```

Comment:
Just wondering, will this be the recommended setup in terms of file structure? although in the end its the integrators choice, I think at least having a "recommended" way of doing it is good! 👍

Reply:
Honestly in a perfect world this should be in its own module, we don't have a `yarn new` template for that yet so this was the simplest way to move forward. Based on the need from the Community for these docs to be updated sooner than later I felt like this was the best middle ground.

Reply:
Clear! was just checking, I'm also fine with this :)